CONTRIBUTING.md

```diff
@@ -50,6 +50,21 @@ should update snapshots by `npm test -- -u`. Then add it in commit.
 npm run docs
 ```
 
+#### Create new components
+
+``` bash
+# Generate a new component in the `./packages/**/` directory
+npm run new-component "[Name]" "[Description]" "[Atoms / Molecules / Organisms]"
+```
+
+Example: Creating a new component 'My Component' in the Organism folder:
+
+``` bash
+npm run new-component "My Component" "A demonstration component." "Organisms"
```

Comment:
Hey @alan-cole I feel this may need to be `new-package` not `new-component`. As we don't want one component per package anymore. How do you think?

Reply:
Yep, that's fine. I'll update naming to replace 'component' with 'package'